### ISSUE TITLE
How to use this repository.

### ISSUE THREAD

**Issue by AdamScott121** [PlaceholderGames/2016-Harambe-s-Sterling-Legacy#5]
This issue will have important information about how to use this repository for our game development project.

Repo Manager - Adam

TL;DR: - Too Long, Didn't Read!
1. Each of us have assigned forks. Confine your work there!
2. Do NOT push to master unless agreed by the whole team
3. Praise Harambe for he is eternal.
# 

When working on issues assigned to you please ensure you work from the **FORK** (or branch) assigned to you. As you work please confine all work current issues to your own **FORK** please do NOT push any finished issues to the master until approved by the whole team. 
This is to prevent the master version from getting messed up and causing a whole host of issues.

If you spot any bugs or decide to start working a new feature for the game, please create the appropriate issue and tag it using the whatever labels required. If you would like me to this for you please ask me. It's important we keep track of these things.

Lastly I will working on putting together a sort of git "cheat sheet" that I will upload and attach to this issue. This will have some basic tutorial on how to use git and some important commands/tools.

/**************************************************************************************************************/
RESERVED SPACE

@BojanStankovic  - Suggested to branch from master for each issue and merge when complete.
